helics_apps-pip/setup.py

```diff
@@ -1,40 +1,131 @@
+# -*- coding: utf-8 -*-
 import io
 import os
 import versioneer
 
-from setuptools import setup
+from setuptools import setup, Extension
+from setuptools.command.build_ext import build_ext
+import subprocess
+import sys
+import re
+import tarfile
+import platform
+import shlex
+
+try:
+    from urllib2 import urlopen
+except ImportError:
+    from urllib.request import urlopen
 
 # Get the directory setup.py is located in
 setup_py_dir = os.path.abspath(os.path.dirname(__file__))
 
 # Get the README.md contents
-README_contents = io.open(os.path.join(setup_py_dir, 'README.md'), encoding="utf-8").read()
+README_contents = io.open(os.path.join(setup_py_dir, "README.md"), encoding="utf-8").read()
+
+HELICS_SOURCE = os.path.join(setup_py_dir, "./_source")
+HELICS_VERSION = versioneer.get_version()
```

Comment:
The versioneer version is the Python package version with post/pre tags, and devN for commits made after a tagged/pypi release (e.g. `v2.7.0.post.dev1`). This won't work for downloading a HELICS source tarball.

Reply:
Fixed in https://github.com/GMLC-TDC/helics-packaging/pull/11/commits/e1b79f809026a7ae54534c92f49890f3917e341b.